src/models/flight_control/FGLinearActuator.cpp

```diff
@@ -0,0 +1,257 @@
+    {
+        
+        if (element->FindElement("set")) {
+            isSetProperty = true;
+            setProperty = PropertyManager->CreatePropertyObject<double>(element->FindElementValue("set"));
+        }
+        if (element->FindElement("reset")) {
+            isResetProperty = true;
+            resetProperty = PropertyManager->CreatePropertyObject<double>(element->FindElementValue("reset"));
+        }
+        
+        ptrVersus = nullptr;
+        if (element->FindElement("versus")) {
+            string property_string = element->FindElementValue("versus");
+            ptrVersus = new FGParameterValue(property_string, PropertyManager);
+            double setVersus = ptrVersus->GetValue();
+            if (ptrVersus->IsConstant()) {
+                if (setVersus >= 0.5) {
+                    versus = 1;
+                } else if (setVersus <= -0.5) {
+                    versus = -1;
+                }
+            }
+        } else versus = 0;
+        
+        ptrBias = nullptr;
+        if (element->FindElement("bias")) {
+            string property_string = element->FindElementValue("bias");
+            ptrBias = new FGParameterValue(property_string, PropertyManager);
+            bias = ptrBias->GetValue();
+        } else bias = 0;
+        
+        if (element->FindElement("module")) {
+            module = element->FindElementValueAsNumber("module");
+            if (module < 0) module = 1.0;
+        }
+        if (element->FindElement("hysteresis")) {
+            hysteresis = element->FindElementValueAsNumber("hysteresis");
+            if (hysteresis < 0) hysteresis = 0.0;
+        }
+        
+        if (element->FindElement("lag")) {
+            double aLag = element->FindElementValueAsNumber("lag");
+            if (aLag > 0.0) {
+                lag = aLag;
+                double denom = 2.00 + dt*lag;
+                ca = dt * lag / denom;
+                cb = (2.00 - dt * lag) / denom;
+            }
+        }
+        
+        if (element->FindElement("rate")) {
+            double aRate = element->FindElementValueAsNumber("rate");
+            if (aRate <= 0 or aRate > 1.0) aRate = 0.5;
+            rate = aRate;
+        }
+        
+        if (element->FindElement("gain")) gain = element->FindElementValueAsNumber("gain");
+        
+        FGFCSComponent::bind();
+        
+        Debug(0);
+    }
+    
+    // %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    
+    FGLinear_Actuator::~FGLinear_Actuator()
+    {
+        Debug(1);
+    }
+    
+    // %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    
+    bool FGLinear_Actuator::Run(void )
+    {
+        
+        if (isSetProperty) set = abs(setProperty) >= 0.5;
+        if (isResetProperty) reset = abs(resetProperty) >= 0.5;
+        
+        if (reset) {
+            input_mem = 0.0;
```

Comment:
Do not mix underscore separated names with CamelCase names. Use `inputMem` or `InputMem` instead.